packages/client/src/components/ui/SidebarAvatar/SidebarAvatar.tsx

```diff
@@ -1,19 +1,38 @@
-import React, { memo } from 'react'
+import React, { memo, useEffect } from 'react'
 import { Wrapper, Avatar, UserName, UserEmail } from './StyledComponents'
-import avatar from '@/image/avatar.jpg'
+import avatar from '@/image/avatar.png'
 import { DarkModeType } from '@/types'
 import { useCustomIntl } from '@/hooks'
+import { useGetUserInfoMutation } from '@/store'
+import { API } from '@/utils'
 
 export const SidebarAvatar = memo(({ darkMode }: DarkModeType) => {
+  const [trigger, { data, isSuccess }] = useGetUserInfoMutation()
+
+  useEffect(() => {
+    trigger(false)
+  }, [])
+
   return (
     <Wrapper darkMode={darkMode}>
-      <Avatar src={avatar} alt={useCustomIntl('AVATAR_USER_ALT')} />
-      <div>
-        <UserName darkMode={darkMode}>
-          {useCustomIntl('USER_NAME_TEST_DATA')}
-        </UserName>
-        <UserEmail>{useCustomIntl('USER_EMAIL_TEST_DATA')}</UserEmail>
-      </div>
+      {isSuccess && (
+        <>
+          <Avatar
+            src={
+              data && data.avatar
+                ? API.API_BASE_URL + API.RESOURCES + data.avatar
+                : avatar
+            }
+            alt={useCustomIntl('AVATAR_USER_ALT')}
+          />
+          <div>
+            <UserName darkMode={darkMode}>
+              {`${data!.first_name} ${data!.second_name}`}
```

Comment:
```suggestion
              {`${data?.first_name} ${data?.second_name}`}
```
Если data не будет, то в рантайме будет ошибка